### ISSUE TITLE
maybe a bug

### ISSUE THREAD

**Issue by shuo-02** [archon159/RAFT#7]
What is the significance of line 163 in exp_long_term_forecasting.py where test_loss = self.vali(test_data, test_loader, criterion) is written? Moreover, test_data is used here, but the mode is 'valid'. Using the index of test_data to index retrieval_dict['valid'] is obviously incorrect.

**Comment by archon159** [comment]
Line 163 is used to track the test dataset loss during training and does not affect the model’s training process.
For evaluation, we used a separate script to extract predictions and compute the MSE.
You may remove or modify that line as needed for your implementation.
